segway/_util.py

```diff
@@ -189,11 +189,25 @@ def get_col_index(chromosome, trackname):
 
 
 def get_label_color(label):
-    color = SCHEME[label % NUM_COLORS]
+    color = SCHEME[hash_label(label) % NUM_COLORS]
 
     return ",".join(map(str, color))
 
 
+def hash_label(label):
+    """
+    If label is an integer or a string containing an integer, return it as 
+    an integer.
+    Otherwise, hash the label string as the sum of the Unicode 
+    representations of its characters.
+    Used because hash() does not replicate across Python runs (multiple 
```

Comment:
I would not put implementation reasoning in a doc string. A comment would be more appropriate.

Reply:
This isn't really needed given we're removing this coloring implementation in the other branch; nonetheless, I made the change.